app/shared/video-player/src/commonMain/kotlin/ui/progress/PlayerControllerBar.kt

```diff
@@ -114,6 +135,107 @@ object PlayerControllerDefaults {
         }
     }
 
+    @Composable
+    fun AudioIcon(
+        volume: Float,
+        isMute: Boolean,
+        onClick: () -> Unit,
+        onchange: (Float) -> Unit,
+        controllerState: VideoControllerState,
+        modifier: Modifier = Modifier,
+        interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    ) {
+        val hoverInteraction = remember { MutableInteractionSource() }
+        val isHovered by hoverInteraction.collectIsHoveredAsState()
+        val audioIconRequester = remember { Any() }
+        LaunchedEffect(isHovered) {
+            controllerState.setRequestAlwaysOn(audioIconRequester, isHovered)
+        }
+        Box(
+            modifier = modifier.hoverable(hoverInteraction)
+                .clip(CircleShape)
+                .ifThen(isHovered) {
+                    background(aniColorScheme().surface.copy(alpha = .1f))
+                },
+            contentAlignment = Alignment.BottomCenter,
+        ) {
+            val iconButton = @Composable {
+                IconButton(
+                    onClick = onClick,
+                ) {
+                    when {
+                        isMute -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeOff, contentDescription = "静音")
+                        }
+
+                        volume < 0.33f -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeMute, contentDescription = "音量")
+                        }
+
+                        volume < 0.66f -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeDown, contentDescription = "音量")
+                        }
+
+                        else -> {
+                            Icon(Icons.AutoMirrored.Rounded.VolumeUp, contentDescription = "音量")
+                        }
+                    }
+                }
+            }
+
+            iconButton()
+
+            Popup(
+                alignment = Alignment.BottomCenter,
+            ) {
+                Box(
+                    modifier = Modifier
+                        .hoverable(hoverInteraction)
+                        .clip(shape = CircleShape)
+                        .background(aniColorScheme().surface),
```

Comment:
```suggestion
                        .background(MaterialTheme.colorScheme.surface),
```

这种情况下实际上应该把 Box 换为 Surface, Surface 自带 surface color